### PR TITLE
Recursor fixes

### PR DESCRIPTION
**Pull request by cmouse**
Fix small things in recursor code.

### CODE REVIEW

**Comment by cmouse** [comment]
@ahu can you look at this as it touches validate.cc

**Comment by cmouse** [comment]
Please restart travis. Was bogus dnssec sig in gcc, not in clang.

**Comment by cmouse** [comment]
merge conflict fixed, please merge?

File: pdns/validate.cc [hunk 1]
```diff
@@ -103,10 +104,10 @@ void validateWithKeySet(const cspmap_t& rrsets, cspmap_t& validated, const keyse
 	  if(signature->d_siginception < now && signature->d_sigexpire > now)
 	    isValid = DNSCryptoKeyEngine::makeFromPublicKeyString(l.d_algorithm, l.d_key)->verify(msg, signature->d_signature);
 	  else
-	    ; // cerr<<"signature is expired/not yet valid ";
+	    DLOG(cerr<<"signature is expired/not yet valid"<<endl);
```
Comment (ahupowerdns) [inline pdns/validate.cc]: DLOG really wants to be { within braces }

**Pull request merged**